frontend/src/app/modules/header/header-item/header-item.component.sass

```
@@ -125,3 +122,10 @@ label:hover
         label
             display: none
 
+.team-name, label
+    overflow: hidden
+    text-overflow: ellipsis
+
+label
+    width: 150px
+    display: inline-block
```

```suggestion
label
    max-width: 150px
    display: inline-block
```

If team name is short - the button containing this label would be unnecessarily large. Isn't it better to use `max-width` instead?